Make.inc

```diff
@@ -641,7 +641,7 @@ else
 SANITIZE_OPTS := -fsanitize=address -mllvm -asan-stack=0
 SANITIZE_LDFLAGS := -fsanitize=address
 endif
-JCXXFLAGS += $(SANITIZE_OPTS)
+JCXXFLAGS += $(SANITIZE_OPTS) -DNDEBUG
```

Comment:
I get ```/julia/usr/lib/libjulia.so: undefined reference to `llvm::cfg::Update<llvm::BasicBlock*>::dump() const'``` without this flag. https://github.com/JuliaLang/julia/issues/35338#issuecomment-607557859

I stole this flag from: https://github.com/imageworks/OpenShadingLanguage/issues/1069#issuecomment-571130730

Reply:
This also disables assertions. We should figure out what calls this in ASAN mode without enabling it.

Reply:
(And then patch LLVM to fix this)